README.md

```diff
@@ -1,24 +1,159 @@
-# README
+# Budget App<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [Blog Hub](#tdd-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 TDD-Project ](#-tdd-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Run tests](#run-tests)
+    - [Run console](#run-console)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Budget App <a name="about-project"></a>
 
-* Database creation
+**Budget App** is a mobile web application for efficient budget management. Easily track your expenses and spending habits by categorizing transactions. Gain insights into your financial trends and stay in control of your budget.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>Rails</li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"><a>
 
-* ...
+- [Login-page]
+- [Category-page]
+- [Transaction-page]
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://rails-budget-app-9a1r.onrender.com/)
+
```

Comment:
- [ ] Please make sure that your deployed version of your app works fine as expected: Because currently if I try to get the login page, it returns an error showing in the image below that shows your app was not appropriately deployed.
  
  ![image](https://github.com/ezraermy/Budget-App/assets/92303891/7c60aa3c-a251-46db-bcbc-8e072c4dfa2e)

Reply:
@Owusu-Desmond Thanks for the feedback. The picture below shows the log of the error I am encountering at the moment. I've tried every possible solution that i know. If you have any idea how to fix this please recommend me.
![Screenshot from 2023-09-14 03-30-23](https://github.com/ezraermy/Budget-App/assets/35451409/61c80fa5-dcc1-4bd6-81cb-b8ff0cb34d9e)